src/App.js

```diff
@@ -1,12 +1,54 @@
-import React from 'react';
+import React, {
+    useEffect,
+    useState
+} from 'react';
 import './App.css';
+import axios
+    from "axios";
+import PokemonCard from './components/PokemonCard'
+import logo from './assets/logo.jpg';
 
 function App() {
-  return (
-    <div>
-      Begin hier met de opdracht!
-    </div>
+    const [pokemon, setPokemon] = useState([]);
```

Comment:
Zodra de setter aangeroepen is, dan is `pokemon` niet meer van het type array. Ik zou daarom de default waarde `undefined` maken
```suggestion
    const [pokemon, setPokemon] = useState();
```